src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/AuthDAO.scala

```diff
@@ -4,35 +4,35 @@ import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevels.WorkspaceAccess
 import org.broadinstitute.dsde.rawls.model._
```

Comment:
Do we have plans to make our Component classes adhere to these traits or should these be removed too?